sql/core/src/main/scala/org/apache/spark/sql/execution/script/ScriptTransformbase.scala

```diff
@@ -0,0 +1,167 @@
+/*
```

Comment:
nit filename: `base` -> `Base`. btw, why did you made a subdir for this single file?

Reply:
> nit filename: `base` -> `Base`. btw, why did you made a subdir `script` for this single file?

Since in origin pr, there is a SparkScriptTransformationExec and remove it now will added in next pr.
So keep it or remove package?

Reply:
Yea its okay to remove it cuz I cannot find any strong reason to create the package.

Reply:
> Yea its okay to remove it cuz I cannot find any strong reason to create the package.

Yea, done.